src/DataContainer/Theme.php

```diff
@@ -39,8 +39,6 @@ public function __construct()
      */
     public function checkPermission(): void
     {
-        parent::checkPermission();
```

Comment:
The `checkPermission` method have been removed in Contao 5.x if I'm not mistaken, SG 1.0.x only supports Contao 4.13.
So this change should not affect SG1.0.x.
Handling this permission removal is done in SG 1.1, see : https://github.com/Web-Ex-Machina/contao-smartgear/pull/309/commits/4afddd4bba64e554728d9f1e307eff7b9d232a22

Reply:
See https://github.com/contao/contao/commit/68b169eca43e4fc7ef3dddc7336b0c84905dec92

Reply:
Didn't realize the [5.3.0-RC1](https://github.com/contao/contao/releases/tag/5.3.0-RC1) tag was covering the 4.13.36 version 🤦 
For a better retro-compatibility (in case we ever need it), we should do something along
```php
public function checkPermission(): void
{
    if (method_exists(\tl_theme::class, 'checkPermission')) {
        parent::checkPermission();
    }
```

Reply:
Done in 5013223e4d0160d062f5146acaacf3fb70ae8553